lib/reinteractive/style/version.rb

```diff
@@ -1,5 +1,5 @@
 module Reinteractive
   module Style
-    VERSION = "0.2.5".freeze
+    VERSION = "0.2.6".freeze
```

Comment:
Please don't update the version number. This is done as part of creating a new release.